src/tracking/tracking-queue.js

```diff
@@ -41,10 +45,11 @@
 		push: push
 	};
 
-	M.geoRequiresConsent = instance.geoRequiresTrackingConsent();
+	M.geoRequiresConsent = instances.gdpr.geoRequiresTrackingConsent();
+	M.geoRequiresSignal = instances.ccpa.geoRequiresUserSignal();
 	M.resetTrackingOptIn = function () {
-		instance.reset();
+		instances.gdpr.reset();
 	};
 	// TODO: Remove this flag once we fully switch to CMP from TrackingOptIn - ADEN-7432
```

Comment:
This todo can be addressed now, what do you think?

Reply:
Yep, line removed.